dagshub/common/cli.py

```diff
@@ -49,5 +54,71 @@ def login(ctx, token, host):
         print("OAuth token added")
 
 
+def validate_repo(ctx, param, value):
+    parts = value.split("/")
+    if len(parts) != 2:
+        raise click.BadParameter("repo needs to be in the format <repo-owner>/<repo-name>")
+    return tuple(parts)
+
+
+def validate_user(ctx, param, value):
+    if value is None:
+        return None, None
+    parts = value.split(":")
+    if len(parts) != 2:
+        raise click.BadParameter("user needs to be in the format <username>:<password>")
+    return tuple(parts)
+
+
+def to_log_level(verbosity):
+    if verbosity == 0:
+        return logging.ERROR
+    elif verbosity == 1:
+        return logging.WARN
+    elif verbosity == 2:
+        return logging.INFO
+    elif verbosity >= 3:
+        return logging.DEBUG
```

Comment:
Default loglevel for python logging is `logging.WARN`
I feel like hiding warnings behind the verbosity flag is a bad idea and we should do
```suggestion
def to_log_level(verbosity):
    if verbosity == 0:
        return logging.WARN
    elif verbosity == 1:
        return logging.INFO
    elif verbosity >= 2:
        return logging.DEBUG
```